docs/database/readme.md

```diff
@@ -0,0 +1,159 @@
+# Schema
+
+```mermaid
+---
+title: Airneis database model
+---
+
+classDiagram
+
+Categories <|-- Products
+Carts <|-- Status
+Products <|-- Carts
+Users <|-- Addresses
+Users <|-- Carts
+Carts <|-- CartsProducts
+Products <|-- CartsProducts
+Products <|-- ProductImages
+Materials <|-- ProductsMaterials
+Products <|-- ProductsMaterials
+
+class Users {
+    id increments notNull
+    fullName text notNull
+    email text unique notNull
+    isAdmin boolean default false notNull
+    isActive boolean default false notNull
+}
+
+class Categories {
+    id increments notNull
+    name text notNull
+    thumbnailUrl text notNull
+    slug text unique notNull
+    orderOfDisplay number notNull
```

Comment:
What is the use of `orderOfDisplay`?

Reply:
In the back office you should be able to change the order of display of the products. 
So that's why we store it in db.

Reply:
Sounds good